### PR TITLE
Update zh

### PR DESCRIPTION
**Pull request by SettingDust**
Waiting for the new options can be translated

### CODE REVIEW

File: app/src/main/res/values-zh/strings.xml [hunk 1]
```diff
@@ -100,6 +100,6 @@
     <string name="toast_newFilterSelect">过滤选择更新，清除图像缓存</string>
     <string name="toast_newTag">搜索标签更新，清除图像缓存</string>
     <string name="toast_newUpdateMode">模式更新，清除图像缓存</string>
-    <string name="dialog_logoutConfirm">您要登出吗？</string>
-    <string name="action_sign_in_refresh">登录用Refresh Token</string>
+    <string name="dialog_logoutConfirm">您确定要登出吗？</string>
+    <string name="action_sign_in_refresh">刷新登录令牌</string>
```
Comment (SettingDust) [inline app/src/main/res/values-zh/strings.xml:104]: What is the `Login using refresh token`? Refresh the pixiv token?

Comment (linsui) [reply app/src/main/res/values-zh/strings.xml:104]: See https://github.com/yellowbluesky/PixivforMuzei3/issues/141#issuecomment-778557052 . I guess it means 通过刷新令牌登录 .

Comment (SettingDust) [reply app/src/main/res/values-zh/strings.xml:104]: @yellowbluesky Is it means to `input a new token`?

Comment (yellowbluesky) [reply app/src/main/res/values-zh/strings.xml:104]: Sorry, those strings can all be taken out, they're not longer necessary

**Pull request closed without merging**